src/pynwb/misc.py

```diff
@@ -158,7 +164,7 @@ class Units(DynamicTable):
             {'name': 'waveform_unit', 'type': str,
              'doc': 'Unit of measurement of the waveform means', 'default': 'volts'},
             {'name': 'resolution', 'type': 'float',
-             'doc': 'The smallest possible difference between two spike times', 'default': None},
```

Comment:
Note: with trailing commas, when you add an element to this list and view the diff in a version control system like Git, then only the new element will show up as changed, rather than both the new element and the previous line which just has a new comma (see example with your addition of the 'waveforms' column. PEP8 allows this. I recently discovered this and prefer it. You can change it if you want. Just wanted to mention why it might have been added.

Reply:
hmm I see. It looks like a mistake but I can see why you like it. I guess I'm split on it.